docs/about.md

```diff
@@ -0,0 +1,3 @@
+# PhEval - Phenotypic Inference Evaluation Framework
+
+There is currently no empirical framework to evaluate the performance of phenotype matching and prioritization tools, much needed to guide tuning for cross species inference. Many algorithms are evaluated using simulations, which may fail to capture real-world scenarios. This gap presents a number of problems: it is difficult to optimize algorithms if we do not know which choices lead to better results; performance may be sensitive to factors that are subject to change, such as ontology structure or annotation completeness. We will develop a modular Phenotypic Inference Evaluation Framework, PhEval and use it to optimize our own algorithms, as well as deliver it as a community resource.
```

Comment:
```suggestion
Many variant prioritization tools (such as [Exomiser](https://exomiser.readthedocs.io/) and other computational approaches rely on ontologies and phenotype matching, sometimes involving complex processes such as cross-species inference. The performance of such tools is exceedingly hard to evaluate, because of the many factors involved: changes to the structure of the ontology, cross-species mappings and semantic similarity algorithms can have significant consequences. Furthermore, the lack of suitable real-world problems/corpora lead to the situation that many algorithms are evaluated using simulations, which may fail to capture real-world scenarios. The lack of an evaluation framework that enables the study of effects on data and knowledge inputs on real world problems makes it difficult to optimize algorithms. To this end we are developing a modular Phenotypic Inference Evaluation Framework (PhEval) which is delivered it as a community resource.
```